Userland/Libraries/LibGfx/Color.h

```diff
@@ -237,15 +237,32 @@ class Color {
 #endif
     }
 
-    Color mixed_with(Color other, float weight) const;
+    Color mixed_with(Color other, float weight) const
```

Comment:
Fwiw, I think both of these methods should get an `ALWAYS_INLINE` since they're prone to be used in tight loops.